### PR TITLE
version 0.4

### PR DESCRIPTION
**Pull request by stuartnelson3**
it's been a long time since a release. it would be good to get these changes out, and to have a binary created by a recent version of go.

@SuperQ

### CODE REVIEW

**Comment by stuartnelson3** [comment]
or @juliusv , seems he's the maintainer :)

**Comment by SuperQ** [comment]
LGTM after https://github.com/prometheus/collectd_exporter/pull/55.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.4.0 / 2018-01-22
+
+* [FEATURE] Support collectd types.db file (#35)
+* [FEATURE] Set receive buffer on UDP socket receiving collectd binary data (#36)
+* [CHANGE] Append `_total` to metric name of collectd Counter and Derive types (#41)
+* [CHANGE] Update flag invocation (#47)
+* [CHANGE] Sanitize metric name output (#51)
```
Comment (SuperQ) [inline CHANGELOG.md]: I would call this `BUGFIX`.

**Comment by juliusv** [comment]
:+1: Thanks

**Pull request merged**